tests/lib/__init__.py

```diff
@@ -798,16 +796,16 @@ def prefix_match(path: str, prefix_path: StrPath) -> bool:
         return path.startswith(prefix)
 
     start_keys = {
-        k for k in start.keys() if not any([prefix_match(k, i) for i in ignore])
+        k for k in start.keys() if not any(prefix_match(k, i) for i in ignore)
     }
-    end_keys = {k for k in end.keys() if not any([prefix_match(k, i) for i in ignore])}
+    end_keys = {k for k in end.keys() if not any(prefix_match(k, i) for i in ignore)}
```

Comment:
```suggestion
    end_keys = {k for k in end if not any(prefix_match(k, i) for i in ignore)}
```

And other `keys()` calls in this file? Not entirely sure.